presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -674,6 +674,30 @@ is evaluated after the ``OFFSET`` clause::
      4
     (2 rows)
 
+For the ``FETCH FIRST`` clause, the argument ``ONLY`` or ``WITH TIES``
+controls which rows are included in the result set. If the argument ``ONLY``
+is specified, the result set is limited to the exact number of leading rows
+determined by the count. If the argument ``WITH TIES`` is specified, the result set
+consists of the same set of leading rows and of all the rows with the same rank
```

Comment:
Instead of rank, maybe use the term "peer group", as that's similar to how we explain window functions with ORDER BY.

One thing that's a little confusing is that the concept of "rank" (or "peer group") is not described until the next paragraph where it relates WITH TIES to the ORDER BY clause. Maybe fold the next paragraph into this sentence and say something like: "If WITH TIES is specified, the result set consists of ... and all of the rows in the same peer group as defined by the ordering defined by the ORDER BY clause. The ORDER BY must be present in this case".